teachertool/src/components/HeaderBar.tsx

```diff
@@ -73,20 +77,66 @@ export const HeaderBar: React.FC<HeaderBarProps> = () => {
         );
     };
 
-    const onHomeClicked = () => {
```

Comment:
Do we want to remove this?

Reply:
Yeah, we're not using it anymore. We can add it back if we end up needing the functionality again, but since I'm not confident that we will, I didn't want to have the unused code sitting here.

Reply:
If you mean the home button in general, I thought it looked better (and more consistent with out other pages) without it.

Reply:
Hmm, yeah, that's fair. I was thinking it might be nice to have it there for a teacher scenario. Like "oh, I wanted to look at this one thing in the tutorial, let me go back to the home page." However, since we don't have an easy way to _get_ to the eval tool from the home page I can see how it wouldn't be needed.